roles/openstack-network/handlers/main.yml

```diff
@@ -6,6 +6,19 @@
 - name: enable ip forwarding
   sysctl: name="net.ipv4.ip_forward" value=1 sysctl_set=yes state=present reload=yes
 
+- name: persist exclude ipv4 masquerade
+  lineinfile:
+    dest: /etc/sysconfig/iptables
+    state: present
+    regexp: "-A POSTROUTING -s {{ item.source }} -d {{ item.destination }} -j MASQUERADE"
+    insertafter: EOF
+    line: "*nat\n-A POSTROUTING -s {{ item.source }} -d {{ item.destination }} -j MASQUERADE\nCOMMIT"
```

Comment:
this will not work. Did we try it out?
There will be 2 nat entry blocks in /etc/sysconfig/iptables

Also, this and next one seems to negative each other.